packages/dashboard/src/components/fileUpload/stories/index.js

```diff
@@ -32,23 +30,31 @@ const Container = styled.div`
 export default {
   title: 'Dashboard/Components/FileUpload',
   component: FileUpload,
+  args: {
+    label: 'Upload',
+    isLoading: false,
+    isMultiple: true,
```

Comment:
💭 I think this control doesn't do anything in the story because no uploads are shown